shipment/buyer/receive/README.md

```diff
@@ -30,7 +35,8 @@ The shipment module is under development. The API and documentation may change.
 * `trackingNumber`: the tracking number of the shipment as provided by the carrier or courier
 * `supplierShipment`: the supplier side header of the shipment, see [Supplier shipment header](#buyer-shipment-header)
 * `buyerShipment`: the buyer side header part of the shipment, see [Buyer shipment header](#buyer-shipment-header)
-* `lines`: a list of all the shipment lines, see [Shipment line](#shipment-line)
+* `loadCarriers`: a list of all the load carriers in this shipment, each load carrier containing shipment lines, see [Load carrier](#load-carrier)
+* `lines`: a list of all the shipment lines, not loaded in a load carrier, see [Shipment line](#shipment-line)
 * `locations`: the departure, next destination and final destination locations of a shipment, see [Shipment locations](#shipment-locations)
 * `meta`: meta information about the shipment, see [Shipment meta information](#shipment-meta-information)
 
```

Comment:
Supplier shipment header is out dated, missing invoice number, contacts, supplierParty
buyer party also missing buyerparty, contacts, purchaseOrderTerms

Reply:
Fixed